redhat-cop/pelorus
# Deploytime

To test:

1. Check out branch
2. `runhelm.sh`
3. `helm template charts/exporter/ -f exporters/committime/values.yaml --namespace pelorus --set source_ref=deploytime,source_url=https://github.com/etsauer/mdt-quickstart.git | oc apply -f- -n pelorus`
4. `helm template charts/exporter/ -f exporters/deploytime/values.yaml --namespace pelorus --set source_ref=deploytime,source_url=https://github.com/etsauer/mdt-quickstart.git | oc apply -f- -n pelorus`

The dashboard should now be properly populated for all panels and graphs under "Lead Time" and "Deployment Frequency"

resolves #33
